db/migrate/20150724182915_add_subject_and_comment_template_for_email_settings.rb

```diff
@@ -0,0 +1,24 @@
+class AddSubjectAndCommentTemplateForEmailSettings < ActiveRecord::Migration
```

Comment:
Why do you create these in the DB? And not in [app/views/admin/emails/index.html.haml](https://github.com/openSUSE/osem/blob/master/app/views/admin/emails/index.html.haml) like we do for the other templates?